buildscripts/manylinux/README.md

```diff
@@ -1,29 +1,30 @@
-# README: Building manylinux_2_17 Wheels
+# README: Building manylinux Wheels
 
 
-## Build llvmdev packages for manylinux_2_17
+## Build llvmdev packages for manylinux
 
 Run the script below to start docker off building `llvmdev` base from the current state of the source tree:
 
-- 32-bit linux: `./buildscripts/manylinux_2_17/docker_run_x32.sh build_llvmdev.sh`
-- 64-bit linux: `./buildscripts/manylinux_2_17/docker_run_x64.sh build_llvmdev.sh`
-- aarch64 linux: `./buildscripts/manylinux_2_17/docker_run_aaarch64.sh build_llvmdev.sh`
+- x86_64 linux: `./buildscripts/manylinux/docker_run_x64.sh build_llvmdev.sh`
+    - uses manylinux2014 image for glibc 2.17+
```

Comment:
`manylinux2014` is renamed to `manylinux` for "future-proof"  and that `aarch64` is not `manylinux2014` so the directories are renamed to avoid confusion.